### PR TITLE
added GitHub Section - Content

### PR DESCRIPTION
**Pull request by SomeOrdinaryBro**
Will add banner's Later

### CODE REVIEW

File: Before You Start/4. Git & GitHub Mastery.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Following our exploration of Git, we will transition to the realm of GitHub, rep
 | Section            | Description               |
 |--------------------|---------------------------|
 | [1. Git Fundamentals](#git-fundamentals) | 🚀 Learning Git First |
-| [2. GitHub Essentials]() | ✨ GitHub Comes Next |
+| [2. GitHub Essentials](#github-essentials) | ✨ GitHub Comes Next |
 
-##  Git Fundamentals 👀
+# Git Fundamentals
```
Comment (SomeOrdinaryBro) [inline Before You Start/4. Git & GitHub Mastery.md:18]: Will add a banner here - later

File: Before You Start/4. Git & GitHub Mastery.md [hunk 2]
```diff
@@ -336,7 +336,136 @@ So we brought you up with a (🧔🏽) _**Bob &**_ (👨🏽‍🦲) _**John "GI
 | Project Progress           | 🧔🏽**Bob** and 👨🏽‍🦲**John** continue this collaborative process, creating new branches for other features or bug fixes as needed. Git keeps a clear history of who did what and when. |
 | Backup and Recovery        | Git serves as a backup for their project. If anything goes wrong, they can always revert to a previous commit to recover their work. |
 
+<!-- I think this is enough Git as beginners will not be forking and creating PR's, So going to start with "Setting up git with github" below -->
+---
+# GitHub Essentials
```
Comment (SomeOrdinaryBro) [inline Before You Start/4. Git & GitHub Mastery.md:341]: Another Banner here

File: Before You Start/4. Git & GitHub Mastery.md (hunk 1 above)
Comment (mshajid) [reply Before You Start/4. Git & GitHub Mastery.md:18]: alright.

File: Before You Start/4. Git & GitHub Mastery.md (hunk 2 above)
Comment (mshajid) [reply Before You Start/4. Git & GitHub Mastery.md:341]: Yes.

**Pull request merged**